### DISCUSSION TITLE
Why flash.nvim for neovim-vscode is laggy?

### DISCUSSION THREAD

**Discussion by qlibp** [vscode-neovim/vscode-neovim#2342] [Q&A] [answered]
flash.nvim is amazing, but I don't quite understand why vscode-neovim is laggy with flash.nvim. I just install 2 plugins with nixvim

Anyone could help me to optimize the speed?

**Comment by xiyaowong** [comment]
Could you provide more detailed information? For example, the text content or a screen recording.

**Reply by qlibp** [reply, marked as answer]
I guess it's a vscode plugin problem... The git plugin is too slow that it makes flash rendering lags. I guess it's solved
